english/millenium-falcon/flyaway.md

```diff
@@ -1,3 +1,7 @@
 Suddenly you encounter a giant flying ship lying on the side, and you draw closer your heart starts beating fast. 
 Wait, could it be? Could it really be the Millenium Falcon? As you pinch yourself and realize you're not dreaming, 
 you now enter this giant ship and turn on the controls ! Fly way, you lucky brat, fly away !
+
+"Hey Chewy, who's flying this infernal machine?" You suddenly hear being asked from one of the dark corridors off the cockpit. You're not alone!
+
+You hear footsteps coming your way, you must decide to [hide inside the doorway](hansolo/hansolo.md) and evade detection or [shoot first](https://en.wikipedia.org/wiki/Han_shot_first)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.